### ISSUE TITLE
Incorrectly fails comparison of previous reading

### ISSUE THREAD

**Issue by tommyjlong** [tommyjlong/Itron-Analog-Gas-Meter-Sensor#1]
After running several days, a couple of HA Energy hour intervals were missing.  Syslog showed:
```
Feb 13 00:45:11 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Starting Gas Meter Analyzer
Feb 13 00:48:30 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Meter value is 8686.90

Feb 13 01:45:11 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Starting Gas Meter Analyzer
Feb 13 01:48:29 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Meter value is 8686.95
Feb 13 01:48:29 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py WARNING  - Previously read value > just read value. Not using meter value.

Feb 13 02:48:32 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Meter value is 8686.95
Feb 13 02:48:32 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py WARNING  - Previously read value > just read value. Not using meter value.

Feb 13 03:45:10 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Starting Gas Meter Analyzer
Feb 13 03:48:38 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Meter value is 8687.00

Feb 13 04:45:12 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Starting Gas Meter Analyzer
Feb 13 04:48:30 Ubuntu-ThinkCentre-A52 gasmeter_analyzer.py INFO     - Meter value is 8687.11
```
The logs show the gas meter's values were never decreasing, yet there is a log statement saying previous is greater than just read value.

**Comment by tommyjlong** [comment]
There are some known issues with python's round(), which may or may not be the problem. There are a few things in play:
* round() with floating point
* log statements formatting the floating point %.2f
* writing and reading from the last_read.txt file and conversion from float to string and string to float.

Will implement a couple of changes:
* use numpy.round() instead of round()
* When doing the comparison of previous to current, will do the comparison using ints.  The string that gets written to/ last_read.txt will be an int that preserves the floating point to 2 decimals by multiplying by 100 and then converting to an int.  When doing the comparison of current vs. previous, will temporarily convert the current by multiplying by 100 and converting to an int and will then do the comparison int to int.

**Comment by tommyjlong** [comment]
Will try a fix with commit `13ba694`
